docs/specification.rst

```diff
@@ -1,668 +1,481 @@
+definition of  ``Any`` doubling for both compatability and uncertainty creates
+a situation where it cannot be excluded from such an intersection, and that
+none of the possible interpretations of it are likely to be satisfying for all
+users.
+
+There are a few potential unordered interpretations of ``Any & T``
+(for this section, where T is a non-gradual type).
+The one which was found to be most consistent with existing definitions in the
+type system could not reduce this to either ``Any`` or ``T``, but would still,
+to users of a type ``Any & T`` be identical to ``Any``. This would lead to a
+significant increases in false negatives interacting with gradual typing.
+
+Banning ``Any`` from intersections to prevent this edge case would create
+significant complications for untyped imports, unbound type variables, and the
+general purpose intent of ``Any`` as a means of compatability in the framework
+of gradual typing.
+
+Other considered ways of resolving this in particular was to reword ``Any``'s
+compatability to be more flexible, and treat it as always yielding to a known
+implementation; However, this would increase the complexity of ``Any``, as well
+as create situations where diamond patterns *may* have been resolved, but the
+type system would not know, leading to increased un-typable false-positives.
+
+Choosing to err on the side of False positives would be better for those
+wanting the immediate feedbacks on type safety in an IDE that many users have
+attributed to productiveness.
+
+Choosing to err on the side of false negatives would be more in line with the
+definition of compatability provided by ``Any``
+
+Since an unordered intersection can create an erosion of the barrier between
+typed an untyped code, no longer coercing from one to another, but being
+possible to mix and match, it is predictable that making a decision either way
+on this would to lead to increased friction between typed and untyped code, and
+increase pressure to more fully type code or to treat code that still can't be
+expressed by the type, but which needed to interact with typed code as taboo.
+
+The ordering allows expressing preferring either the False positives for the
+implied possible diamond pattern with untyped things (``T & Any``) and a narrow
+remedy for it (``P & T & Any``) where ``P`` is a protocol expressing how the
+diamond pattern was actually resolved.
+
+The ordering also allows expressing prefering not to get warnings for the
+implied possible diamond pattern: (``Any & T``)
+
+Neither of these provide warnings for things not provided by ``T``, the scope
+of the ordering mattering is only in the overlap.
+
+(TODO add table comparing effects of each option in each of meaningful cases, highlighting the equivalence to unordered in most cases)
+
+Backwards Compatibility
+=======================
+
+This PEP expands the allowed use of ``Self`` to better handle mixins,
+this change is not done in a backwards incompatible manner.

```

Comment:
Yeah this part I don't understand really - appreciate there is an example above, but how can classes be intersected with `Self`? Isn't `Self` just an instance of whatever it's containing class is?

Reply:
This part needs more added to explain still, haven't been adding much to the draft since noticing we have an underlying issue with unclear subtyping in the specification.